<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagEditor</class>
 <widget class="QWidget" name="TagEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>221</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="tagEditorLayout">
     <item row="0" column="3" colspan="7">
      <widget class="QLabel" name="fileNameLabel">
       <property name="text">
        <string>F&amp;ile name:</string>
       </property>
       <property name="buddy">
        <cstring>fileNameBox</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="trackLabel">
       <property name="text">
        <string>T&amp;rack:</string>
       </property>
       <property name="buddy">
        <cstring>trackSpin</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="artistNameLabel">
       <property name="text">
        <string>&amp;Artist name:</string>
       </property>
       <property name="buddy">
        <cstring>artistNameBox</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="albumNameLabel">
       <property name="text">
        <string>Album &amp;name:</string>
       </property>
       <property name="buddy">
        <cstring>albumNameBox</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="genreLabel">
       <property name="text">
        <string>&amp;Genre:</string>
       </property>
       <property name="buddy">
        <cstring>genreBox</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="3" colspan="7">
      <widget class="KLineEdit" name="fileNameBox"/>
     </item>
     <item row="2" column="7">
      <widget class="QLabel" name="yearLabel">
       <property name="text">
        <string>&amp;Year:</string>
       </property>
       <property name="buddy">
        <cstring>yearSpin</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="KLineEdit" name="lengthBox">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="8">
      <widget class="KLineEdit" name="bitrateBox">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="lengthLabel">
       <property name="text">
        <string>&amp;Length:</string>
       </property>
       <property name="buddy">
        <cstring>lengthBox</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QLabel" name="bitrateLabel">
       <property name="text">
        <string>&amp;Bitrate:</string>
       </property>
       <property name="buddy">
        <cstring>bitrateBox</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="3" colspan="2">
      <widget class="QLabel" name="commentLabel">
       <property name="text">
        <string>&amp;Comment:</string>
       </property>
       <property name="buddy">
        <cstring>commentBox</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="3" rowspan="4" colspan="7">
      <widget class="KTextEdit" name="commentBox"/>
     </item>
     <item row="8" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="KComboBox" name="albumNameBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="KComboBox" name="genreBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="KComboBox" name="artistNameBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="KIntSpinBox" name="trackSpin">
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="2" column="8">
      <widget class="KIntSpinBox" name="yearSpin">
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="8">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="trackNameLabel">
       <property name="text">
        <string>Trac&amp;k name:</string>
       </property>
       <property name="buddy">
        <cstring>trackNameBox</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="artistNameEnable">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="KLineEdit" name="trackNameBox"/>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="trackNameEnable">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="albumNameEnable">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="genreEnable">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QCheckBox" name="trackEnable">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item row="2" column="9">
      <widget class="QCheckBox" name="yearEnable">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5" colspan="5">
      <widget class="QCheckBox" name="commentEnable">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6" rowspan="2">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>KLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>artistNameEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>artistNameBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trackNameEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>trackNameBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>albumNameEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>albumNameBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>genreEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>genreBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trackEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>trackSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commentEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>commentBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yearEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>yearSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>558</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>510</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
